openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+
+<!-- staff-info__student is necessary to get proper styling for the staff assessment form -->
+<div class="staff__grade__control ui-toggle-visibility is--collapsed staff-info__student">
+    <header class="staff__grade__header ui-toggle-visibility__control">
+        <h3 class="staff__grade__title">
+            <span class="wrapper--copy">
+                <span class="staff__grade__label">{% trans "Instructor Assessment" %}</span>
+            </span>
+        </h3>
+
+        {% block title %}
+        <span class="staff__grade__status">
+          <span class="staff__grade__value">
+            <span class="copy">
+                {% blocktrans with ungraded=staff_assessment_ungraded|stringformat:"s" in_progress=staff_assessment_in_progress|stringformat:"s" %}
+                    {{ ungraded }} Available and {{ in_progress }} Checked Out
+                {% endblocktrans %}
+            </span>
+          </span>
+        </span>
+        {% endblock %}
+    </header>
+
+    <div class="staff__grade__content ui-toggle-visibility__content">
+        <div class="wrapper--input">
+            <div class="staff__grade__form--error"></div>
+            <div class="staff__grade__form"></div>
```

Comment:
I am concerned how this is for accessibility-- the fact that you have to click on a div to html data from the sever, when is then injected into the DOM. We will have a full accessibility review of staff grading soon, but this part seems particularly troublesome. FYI, @cptvitamin